gogen/goelements.go

```diff
@@ -642,16 +642,24 @@ func (s *GoLangMapper) yangDefaultValueToGo(value string, args resolveTypeArgs,
 		if err != nil {
 			return "", yang.Ynone, err
 		}
+
+		// In some YANG modules the default value could be written as base 16.
+		base := 10
+		if strings.HasPrefix(strings.ToLower(value), "0x") {
```

Comment:
(optional) Would be helpful if this PR also added support for octal numbers, so "0" as the leading string.